### PR TITLE
Jwt

### PR DESCRIPTION
**Pull request by Thomas-Nigon**

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -9,7 +9,9 @@
   },
   "dependencies": {
     "argon2": "^0.31.2",
+    "axios": "^1.6.5",
```
Comment (Sax593) [inline backend/package.json:12]: axios is a frontend library

File: backend/src/controllers/authControllers.js [hunk 2]
```diff
@@ -1,39 +1,59 @@
...
       return;
     }
     const verified = await argon2.verify(user.password, req.body.data.password);
     if (verified) {
+      console.warn("user verified");
       // Respond with the user and a signed token in JSON format (but without the hashed password)
-      delete user.hashed_password;
-      const token = await jwt.sign(
+      delete user.password;
+      /* console.warn("user verified gg !!!!"); */
```
Comment (Sax593) [inline backend/src/controllers/authControllers.js:23]: Death code

File: backend/src/controllers/authControllers.js [hunk 3]
```diff
@@ -1,39 +1,59 @@
...
         process.env.APP_SECRET,
         {
           expiresIn: "1h",
         }
       );
-      res.json({
-        token,
-        user,
+      res.cookie("access_token", token, {
+        /* sameSite: "None", */
```
Comment (Sax593) [inline backend/src/controllers/authControllers.js:32]: death code

File: backend/src/controllers/authControllers.js
Comment (Sax593) [inline backend/src/controllers/authControllers.js:1]: please delete console.warn

File: backend/src/controllers/userControllers.js [hunk 4]
```diff
@@ -20,12 +42,15 @@ const register = async (req, res, next) => {
   try {
     const user = await tables.user.create(myUser, hashedPassword);
     res.json(user);
+    /*     console.log("created user:", user); */
```
Comment (Sax593) [inline backend/src/controllers/userControllers.js:45]: death code

File: backend/src/router.js [hunk 5]
```diff
@@ -29,18 +31,18 @@ router.get(
...
-router.get("/ustensil", ustensilControllers.browse);
 router.get("/tag", tagControllers.browseByTag);
 router.get("/user", userControllers.browse);
-
 router.post("/adduser", hashPassword, userControllers.register);
 router.post("/login", authControllers.login);
+router.get("/logout", authControllers.logout);
+// router.post("/login", (req,res)=>{
+//   console.log("body",req.body);
+// });
```
Comment (Sax593) [inline backend/src/router.js:41]: death code

File: backend/src/router.js [hunk 6]
```diff
@@ -29,18 +31,18 @@ router.get(
...
 
-router.get("/ingredient", ingredientControllers.browse);
-router.get("/step", stepControllers.browse);
+/* router.get("/ingredient", ingredientControllers.browse); */
+/* router.get("/step", stepControllers.browse); */
```
Comment (Sax593) [inline backend/src/router.js:44]: death code

File: frontend/src/components/Contexts/userContext.jsx [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+  useEffect(() => {
+    setConnection();
+
+    /*  try {
+      fetch(`${import.meta.env.VITE_BACKEND_URL}/api/me`)
+        .then((response) => response.json())
+        .then((data) => console.warn("data du fetch:", data));
+    } catch (error) {
+      console.error(error);
+    } */
```
Comment (Sax593) [inline frontend/src/components/Contexts/userContext.jsx:38]: death code

File: frontend/src/components/LoginForm/LoginForm.jsx [hunk 8]
```diff
@@ -1,17 +1,44 @@
...
+              isLogged: true,
+            }),
+          navigate("/")
+        );
+      /*  .then((response) => {
+          if (response.status === 200) {
+            console.warn("auth after loging:", auth);
+            // navigate("/");
+          } else console.error("Wrong password or email");
+        }); */
```
Comment (Sax593) [inline frontend/src/components/LoginForm/LoginForm.jsx:38]: death code

**Review by Sax593 — CHANGES_REQUESTED** [review]

**Review by NemoOnTheNet — APPROVED** [review]

**Pull request merged**